math/perimeter.cpp

```diff
@@ -102,14 +103,14 @@ T n_polygon_surface_perimeter(T sides, T length) {
 
 /**
  * @brief surface perimeter of a
- * [cylinder](https://en.wikipedia.org/wiki/Cylinder) (2 * radius + 2 * height)
+ * [cylinder](https://en.wikipedia.org/wiki/Cylinder) 2 * (2 * pi * radius + height)
```

Comment:
```suggestion
 * [cylinder](https://en.wikipedia.org/wiki/Cylinder)  \f$2\times\textrm{radius} + 2 \times\textrm{height}\f$
```
please note that the formula implemented in the function should be the same that in the comments.

Reply:
Oops missed that somehow. Will make that change